azerothcore/azerothcore-wotlk
# refactor(Core): apply clang-tidy modernize-raw-string-literal

- Part of https://github.com/azerothcore/azerothcore-wotlk/issues/3820
- Apply `clang-tidy` with `modernize-raw-string-literal`
https://clang.llvm.org/extra/clang-tidy/checks/modernize-raw-string-literal.html

### How to test the changes

- Build with `-DTOOLS=1`
- [Extract the client files](https://www.azerothcore.org/wiki/Extract-Client-Data)
- Run the server
- Access the game
- All good so far? Then this should be good to go

## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
